data/kubevirt-ipam-controller/001-kubevirtipamcontroller.yaml

```diff
@@ -170,8 +170,14 @@ spec:
       control-plane: manager
   template:
     metadata:
+{{ if .IsOpenshift }}
+      annotations:
+        openshift.io/required-scc: "restricted-v2"
+        kubectl.kubernetes.io/default-container: manager
```

Comment:
nit: why include this in the `.IsOpenshift` conditional?  I'd have expected:

```
      annotations:
        kubectl.kubernetes.io/default-container: manager
{{ if .IsOpenshift }}
        openshift.io/required-scc: "restricted-v2"
{{ end }}
```

Reply:
agree,
also here please
https://github.com/kubevirt/cluster-network-addons-operator/pull/1931/files#diff-5cadb26d0d796a5eff24e582b27e36a926bcaa614a8df5fd54d3b52b4d3addd2R24

Reply:
I guess it's possible, I'll try.